contracts/RootChain.sol

```diff
@@ -135,15 +147,22 @@ contract RootChain {
         // Only allow up to CHILD_BLOCK_INTERVAL deposits per child block.
         require(currentDepositBlock < CHILD_BLOCK_INTERVAL);
 
-        bytes32 root = keccak256(msg.sender, address(0), msg.value);
-        uint256 depositBlock = getDepositBlock();
-        childChain[depositBlock] = ChildBlock({
-            root: root,
-            timestamp: block.timestamp
-        });
-        currentDepositBlock = currentDepositBlock.add(1);
+        emitDepositBlock(msg.sender, address(0), msg.value);
```

Comment:
I would rename this function. It looks like a typo of `emit DepositBlock`.

Reply:
done